production/ksonnet/loki/memberlist.libsonnet

```diff
@@ -23,6 +17,11 @@
     // but "primary" KV depends on value of multikv_primary.
     memberlist_ring_enabled: false,
 
+    // Configures the memberlist cluster label. When verification is enabled, a memberlist member rejects any packet or stream
+    // with a mismatching cluster label.
+    memberlist_cluster_label: '',
```

Comment:
I wonder if we should have a default here? Do we know what the behavior is if the verification is enabled but the label is empty (which I believe is the case the way this PR is currently written)

Or maybe we should have this error if it's not specified??

Reply:
I think we can have the default use some reasonable label, like `"loki"` and set verification to false by default, no?

I'm also not sure why we have `$._config.memberlist_<field>` when we later add
```
        cluster_label: $._config.memberlist_cluster_label,
        cluster_label_verification_disabled: $._config.memberlist_cluster_label_verification_disabled,
```
        
        
  It should work, it's just a bit indirect. I'd prefer we set the field itself though.

Reply:
> I think we can have the default use some reasonable label, like "loki" and set verification to false by default, no?
We can't do that either because the default is doing the verification and even if we disable it here older instances would still do the verification causing the same problem.

> I'm also not sure why we have $._config.memberlist_<field> when we later add
I just copied over the jsonnet. But yeah, setting it directly would be better. I will change it.

Reply:
Makes sense, thanks